README.Rmd

```diff
@@ -1,138 +1,98 @@
 ---
-output: html_document
-always_allow_html: yes
+output: github_document
 params:
-   website_url: http://appsilon.com
-   logo_url: https://cdn.rawgit.com/Appsilon/website-cdn/gh-pages/logo-white.png
-   repo_name: shiny.semantic
-   repo_url: https://github.com/Appsilon/shiny.semantic
-   demo_url: https://demo.appsilon.com/shiny-semantic-components/
-   chagelog_url: https://github.com/Appsilon/shiny.semantic/blob/master/CHANGELOG.md
+  website_url: http://appsilon.com
+  logo_url: https://cdn.rawgit.com/Appsilon/website-cdn/gh-pages/logo-white.png
+  repo_name: shiny.semantic
+  repo_url: https://github.com/Appsilon/shiny.semantic
+  demo_url: https://demo.appsilon.ai/shiny-semantic-components/
+  chagelog_url: https://github.com/Appsilon/shiny.semantic/blob/master/CHANGELOG.md
 ---
 
-```{r, echo = FALSE, message = FALSE}
-knitr::opts_chunk$set(
-  collapse = TRUE,
-  comment = "#>",
-  fig.path = "inst/assets/README_files/figure-markdown_github"
-)
-suppressWarnings(library(shiny.semantic))
-```
 
 ```{r, echo = FALSE, message = FALSE}
-source("readme_assets/ui_demo.R")
-
-create_example <- function(code, ...) {
-  unstyled <- deparse(substitute(code), width.cutoff = 20)
-  args <- as.list(unstyled)
-  args$sep <- "\n"
-  args <- as.list(styler::style_text(do.call(paste, args)))
-  args$sep <- "\n"
-  styled <- do.call(paste, args)
-  list(
-    htmlwidget = shiny.semantic::uirender(div(class="demo", code), ...),
-    code_demo = div(class="demo-code",
-      pre(code(class="r", styled))
-    )
-  )
-}
+suppressWarnings(library(shiny.semantic))
+suppressWarnings(library(shiny))
 ```
 
-<link href="http://fonts.googleapis.com/css?family=Maven+Pro:400,700|Inconsolata" rel="stylesheet" type="text/css">
-<link href='docs/style.css' rel='stylesheet' type='text/css'>
 
-`r params$repo_name`
+  `r params$repo_name`
 ===========
 
-<div class="subheader">Semantic UI wrapper for Shiny
+<img src="man/figures/hexsticker.png" align="right" alt="" width="120" />
+
+**Semantic UI wrapper for Shiny**
 
-</br>
+<!-- badges: start -->
+
+[![Travis build status](https://travis-ci.org/Appsilon/shiny.semantic.svg?branch=develop)](https://travis-ci.org/Appsilon/shiny.semantic) 
 
-[![Travis build status](https://travis-ci.org/Appsilon/shiny.semantic.svg?branch=develop)](https://travis-ci.org/Appsilon/shiny.semantic)
 [![codecov](https://codecov.io/gh/Appsilon/shiny.semantic/branch/master/graph/badge.svg)](https://codecov.io/gh/Appsilon/shiny.semantic)
-</div>
 
-<div class="section level2">
+[![cranlogs](https://cranlogs.r-pkg.org/badges/shiny.semantic)](https://CRAN.R-project.org/package=shiny.semantic)
+
+[![total](https://cranlogs.r-pkg.org/badges/grand-total/shiny.semantic)](https://www.rpackages.io/package/shiny.semantic)
+<!-- badges: end -->
+
+
```

Comment:
Can we set badges in one line?

Reply:
It was like that at first, but then it looks quite ugly in docs/index.html for some reason.